wtform_fields.py

```diff
@@ -44,7 +44,6 @@ def validate_password(self, password):
 
 class EnterMessageForm(FlaskForm):
 	""" Enter message form """
-
```

Comment:
I find it really usefull to keep the new lines between comments like this. It makes the code more readable so from an esthetic point of view i would highly recommend you to keep the new lines as they are.